server/routes/routerRequestHandlers/retrieveInductionSchedule.ts

```diff
@@ -10,14 +11,20 @@ const retrieveInductionSchedule = (inductionService: InductionService): RequestH
   return asyncMiddleware(async (req: Request, res: Response, next: NextFunction) => {
     const { prisonNumber } = req.params
 
-    try {
-      // Retrieve the Induction Schedule and store in res.locals
-      res.locals.inductionSchedule = {
-        problemRetrievingData: false,
-        inductionSchedule: await inductionService.getInductionSchedule(prisonNumber, req.user.username),
+    const { activeCaseLoadId } = res.locals.user
```

Comment:
Given that inductionSchedules are a feature toggled thing, we need to wrap this call in a test for the feature toggle 
(I should have done this in my last PR)